wang1kevin1/InstitutoTerra
# Scratch/calculate emissions (#12)

* call working

* in progress working on get distance equation

* test

* call working

* in progress working on get distance equation

* test

* post rebase

* call working

* in progress working on get distance equation

* test

* post rebase

* working API Calls  and error handling done

* removing test files

* added comments

* Cleaning up code, changing file names, etc

* mid changes, needs bug fixes

* distance rounded to hundreth

* Buttons on FLightInfo all work

* Added dynamic button opacity & distance modifier

* App.js preemptively loads font for all children

* added loading screen when retrieving flight info

* New footer cmpnt.  Replaced existing +add to new

* Extended top button bar, added plce for nav btns

* FlightInfo Screen COMPLETED

* CarbonEmissions, working. Adding dynamic UI next

* Edge case: can no longer select negative trees

* Fixing asset file structure

* added a new background for future

* changes to cost for future changes/inflation

* still updating render late

* Years calculates and updates correctly

* Color changes working

* Colors and icons updated

* roundtrip distance passed as well now

* Added NoFlight donations screen, adjusted stack

* Renamed a bunch of screens, working on c02 screen

* First three screen of flight calc workflow done

* file name changes to make flow more understandable

* better file structure

* fixed checkoutwithflight

* Added flight receipt page

* Receipt with flight done

* update

* one more update

* Calculate emissions work flow completed

* fixed carbon emissions non visible on iphone

* changed some stuff for easier UI fixing

* testing emissions page fix

* check size again

* possible fix?

* flight info fixed

* flow with flight UI fixed

* footer changes

* noflight receipt needs some changes.

* using height for carbonemissions page

* kinda fixed receipt noflight

* All checkout flow UI done

* testing dimensions

* Test on small screens

* Hopefully final fix of checkout UI

* changed footer to scale as well

* starting on home page, switching branch after pull

* margin changes please dear god work

* Revert "margin changes please dear god work"

This reverts commit 547215a0a06ba9ae9fc0b89265344b2a8a022e1a.

* try now please

* please try now

* test first page

* FINALLY WORKING UI

* safeareaview removed

Co-authored-by: malcolmneill <44383128+malcolmneill@users.noreply.github.com>
